src/applications/mhv-medical-records/api/MrApi.js

```diff
@@ -38,6 +39,26 @@ export const getLabOrTest = id => {
   });
 };
 
+export const getMhvRadiologyTests = () => {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve(radiologyRecordsMhv);
+    }, 1000);
+  });
+};
+
+export const getMhvRadiologyTest = id => {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve(
+        radiologyRecordsMhv.find(item => {
+          return +item.id === +id;
+        }),
+      );
+    }, 1000);
+  });
+};
```

Comment:
Unfortunately there is no endpoint to get a single radiology test. We always need to pull from the list. So, if the detail record doesn't exist, we'll have to fetch all records, then pick out the one we want.